src/sites/cclw/pages/faq.tsx

```diff
@@ -13,7 +13,7 @@ const FAQ = () => {
 
         {FAQS.map((faq, i) => (
           <Fragment key={faq.title}>
-            <AccordianItem title={faq.title} startOpen={i === 0}>
+            <AccordianItem id={faq.id} title={faq.title} startOpen={i === 0}>
```

Comment:
Hmm ... not all have an `id` what happens in that case - does the attribute still get rendered? Looks like it does in `AccordianItem.tsx` as there are not conditionals around `<div className="mb-4" id={id}>`